contrib/inventory/scaleway.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+'''
+External inventory script for Scaleway
+====================================
+
+Shamelessly copied from an existing inventory script.
+
+This script generates an inventory that Ansible can understand by making API requests to Scaleway API
+
+Requires some python libraries, ensure to have them installed when using this script. (pip install scaleway-sdk https://github.com/scaleway/python-scaleway)
+
+Before using this script you may want to modify scaleway_.ini config file.
+
+This script generates an Ansible hosts file with these host groups:
+
+<hostname>: Defines host itself with Scaleway's hostname as group name.
+<tag>: Contains all hosts which has "<tag>" as tag.
+all: Contains all hosts defined in Scaleway.
+'''
+
+# (c) 2017, Paul B. <paul@bonaud.fr>
+#
+# This file is part of Ansible,
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+import ConfigParser
```

Comment:
This is not python3 compliant.

Reply:
Thanks, i've shamelessly copied from the ec2 inventory: https://github.com/ansible/ansible/blob/devel/contrib/inventory/ec2.py#L313-L316